src/initialization/InitDistribution.cpp

```diff
@@ -310,10 +310,12 @@ namespace impactx
         bool space_charge = false;
         amrex::ParmParse pp_algo("algo");
         pp_algo.queryAdd("space_charge", space_charge);
+        bool csr = false;
+        pp_algo.queryAdd("csr", csr);
 
         // For pure tracking simulations, we keep the particles split equally
         // on all MPI ranks, and ignore spatial "RealBox" extents of grids.
-        if (space_charge) {
+        if (space_charge || csr) {
```

Comment:
This seems to cause an issue in a test (albeit adding this seems sensible...). To check.

Reply:
Actually it is not reasonable, because CSR binning has its separate parameters :D